src/components/Mechanism/WordsOrImage.jsx

```
@@ -242,6 +242,7 @@ const WordsOrImage = ({
             setVoiceAnimate={setVoiceAnimate}
             storyLine={storyLine}
             dontShowListen={type === "image" || isDiscover}
+            isShowCase={isShowCase || isDiscover}
```

Biome

[error] This JSX property is assigned multiple times.

This attribute is assigned again here.


(lint/suspicious/noDuplicateJsxProps)